tests/setup-test.js

```diff
@@ -1,19 +1,71 @@
 'use strict';
 
 var tape = require('tape');
-var server = require('../example/server.js');
+// var Hoek = require('hoek');
+var init = require('../example/server.js').init;
+var tagsPool = require('../example/server.js').tagsPool;
+var tags = require('../example/tags.json');
+var server;
+
+// First check that the DB is empty
+tape('check that the DB is empty', function (t) {
+  tagsPool.connect(function (conErr, client, done) {
+    if (conErr) {
+      t.fail();
+    }
+    t.equal(typeof client, 'object',
+      'We can connect to the db');
+
+    client.query('select * from tags;', function (dbErr, results) {
+      if (dbErr) {
+        return t.fail();
+      }
+
+      t.equal(typeof results.rows, 'object',
+        'we successfully connected to the tags table');
+
+      t.equal(results.rows.length, 0,
+        'The tags table has 0 rows');
+
+      t.end();
+
+      return done();
+    });
+  });
+});
+
+
+// set up server
+tape('set up server', function (t) {
+  init(2000, function (err, newServer) {
+    if (err) {
+      return t.fail();
+    }
+    server = newServer;
+
+    return t.end();
+  });
+});
+
 
 tape('check that plugin is being attached to the request object', function (t) {
   server.inject({
```

Comment:
Do we not have that functionality now? 

The tests define the config for the pg connection, the init function creates the connection unless a test wants to connect to the db without the server being initialised.